src/scss/cdb-components/shapes/arrow.scss

```diff
@@ -0,0 +1,86 @@
+/* SG
+# Shapes/Arrow
+
+Description
+
+```
+<div style="padding: 20px;">
+  <div class="CDB-Shape-Arrow"></div>
+</div>
+<div style="padding: 20px;">
+  <div class="CDB-Shape-Arrow is-blue"></div>
+</div>
+<div style="padding: 20px; background: #2E3C43">
+  <div class="CDB-Shape-Arrow is-white"></div>
+</div>
+```
+*/
+
+@import '../../cdb-variables/colors';
+
+.CDB-Shape-Arrow {
```

Comment:
I think we should make the direction that the arrow is pointing explicit with a class (points-up, points-down, etc.).